client/src/components/Overview/ImageGallery.jsx

```diff
@@ -68,17 +70,19 @@ function ImageGallery(props) {
       setStart(start - 1);
       setEnd(end - 1);
     }
+    setCount(count -1)
   };
 
   const handleThumbnailSelect = (index) => {
     setSelectedThumbnail(index);
+    setCount(index);
   };
 
   if (styles[count]) {
     const photoUrl = styles[count].photos[0].url;
 
     galleryThumbnails = styles.map((style, index) => {
-      const altTxt = `${style.name} ${description.category}`;
+      const altTxt = `${style.name && style.name} ${description && description.category}`;
```

Comment:
Why is this `style.name && style.name`? 

Should it instead be `style && style.name`?